apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ExtensionContext.java

```diff
@@ -28,18 +33,30 @@
  */
 public class ExtensionContext {
 
+    private static final ILog LOGGER = LogManager.getLogger(ExtensionContext.class);
     /**
      * Tracing Mode. If true means represents all spans generated in this context should skip analysis.
      */
     private boolean skipAnalysis;
 
+    /**
+     * The sending timestamp of the exit span.
+     */
+    @Getter
+    @Setter
+    private long sendingTimestamp = 0;
+
     /**
      * Serialize this {@link ExtensionContext} to a {@link String}
      *
      * @return the serialization string.
      */
     String serialize() {
-        return skipAnalysis ? "1" : "0";
+        String res = skipAnalysis ? "1" : "0";
+        if (sendingTimestamp != 0) {
+            res += "-" + sendingTimestamp;
```

Comment:
Maybe we should always serialize this part even `if sendingTimestamp != 0`? The extension fields are order-sensitive, we always need a real value or placeholder, otherwise, if the downstream agent plugin don't have this field but have their own extended fields(now for now, though), it will be trouble, WDYT? @wu-sheng

Reply:
Yes, my previous comment is about don't serialize the 0, but - should be included. 
It seems to be misunderstood.

Reply:
I have similar concerns about this, so the zero is a placeholder before. WDYT? @wu-sheng

Reply:
I don't like `0`, as `0` is a real value, you are going to represent no value. So you could use `0-` as timestamp doesn't exist.